id/community/weblogs/index.md

```diff
@@ -26,6 +26,9 @@ signifikan.
 * [**Ruby Weekly**][ruby-weekly]: Meskipun lebih banyak buletin ketimbang
   blog, Ruby Weekly adalah distilasi dari artikel dan berita Ruby yang
   paling menarik setiap minggu.
+* [**DEV Ruby Tag**][dev-ruby-tag] adalah kumpulan semua *post* dengan penanda
+  Ruby dalam Dev Community. DEV adalah sebuah jaringan dari ribuan pengembang
+  perangkat lunak yang menge-*blog* dan mendiskusikan kode.
```

Comment:
instead of "menge-blog", how about "membuat postingan blog" or "membuat artikel blog"?

Reply:
Actually, I was thinking about that. I prefer the second translation because the first translation contains "postingan" which is not valid in Indonesia. I can't find it on KBBI https://kbbi.kemdikbud.go.id/entri/postingan.

By the way, Ivan Lanin suggested to use "kiriman" instead of "postingan" (https://twitter.com/ivanlanin/status/842698571023503361). The sentence would be "membuat kiriman *blog*".

What do you think, @kuntoaji?

Reply:
Actually, I was thinking about that. I prefer the second translation because the first translation contains "postingan" which is not valid in Bahasa Indonesia. I can't find it on KBBI https://kbbi.kemdikbud.go.id/entri/postingan.

By the way, Ivan Lanin suggested to use "kiriman" instead of "postingan" (https://twitter.com/ivanlanin/status/842698571023503361). The sentence would be "membuat kiriman _blog_".

What do you think, @kuntoaji?

Reply:
ok, LGTM

Reply:
updated ya @kuntoaji